### ISSUE TITLE
Executing rule tslint/use-default-type-parameter fails with TypeError

### ISSUE THREAD

**Issue by maxbogus** [fimbullinter/wotan#569]
Wotan run fails in our app checking with the following error:
`TypeError: Cannot read property 'default' of undefined
    at checkArgsAndParameters `.

**Comment by ajafff** [comment]
That's a bug in TSLint. There's not much this project can do about it.
You can set environment variable `DEBUG="wotan:linter"` to find out which file in your project causes the error. With that information you can probably narrow down the issue and report it upstream.

**Comment by maxbogus** [comment]
I agree that TSLint causes TypeError, but Wotan should not fail in that case. But it does.

I expect in that case, that Wotan will count that error and will continue linting.

**Comment by ajafff** [comment]
> Wotan will count that error and will continue linting.

That's a fair point. Though I prefer halting the process as things might be in a bad state.

I think adding a command line option to configure that behavior is the best way to go here. You could then use  `--rule-crash <severity>` to report exceptions in rules as error, warning or suggestion. Otherwise an exception terminates the process.
The name is up for discussion
